packages/@react-stately/layout/src/TableLayout.ts

```diff
@@ -10,114 +10,72 @@
  * governing permissions and limitations under the License.
  */
 
-import {ColumnSize, TableCollection} from '@react-types/table';
 import {DropTarget, Key} from '@react-types/shared';
 import {getChildNodes} from '@react-stately/collections';
 import {GridNode} from '@react-types/grid';
 import {InvalidationContext, LayoutInfo, Point, Rect, Size} from '@react-stately/virtualizer';
-import {LayoutNode, ListLayout, ListLayoutOptions} from './ListLayout';
+import {LayoutNode, ListLayout, ListLayoutOptions, ListLayoutProps} from './ListLayout';
+import {TableCollection} from '@react-types/table';
 import {TableColumnLayout} from '@react-stately/table';
 
-type TableLayoutOptions<T> = ListLayoutOptions<T> & {
-  columnLayout: TableColumnLayout<T>,
-  initialCollection: TableCollection<T>
+export interface TableLayoutOptions<T> extends ListLayoutOptions<T> {
+  scrollContainer?: 'table' | 'body'
```

Comment:
This is to handle a difference between RSP and RAC. In RSP, only the TableBody scrolls, whereas (so far) in RAC, the whole table scrolls and the column headers have `position: sticky`. This affects the positioning of the body rows, which are relative to the body in RSP, and relative to the whole table in RAC.